cookbook/doctrine/registration_form.rst

```diff
@@ -138,230 +170,143 @@ Next, create the form for the ``User`` model::
+                <route id="user_registration" path="/register">
+                    <default key="_controller">AppBundle:Registration:register</default>
+                </route>
+            </routes>
 
-        account_create:
-            path:     /register/create
-            defaults: { _controller: AcmeAccountBundle:Account:create }
+        .. code-block:: php
 
-    .. code-block:: xml
+            // app/config/routing.php
+            use Symfony\Component\Routing\RouteCollection;
+            use Symfony\Component\Routing\Route;
 
-        <!-- src/Acme/AccountBundle/Resources/config/routing.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <routes xmlns="http://symfony.com/schema/routing"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+            $collection = new RouteCollection();
+            $collection->add('user_registration', new Route('/register', array(
+                '_controller' => 'AppBundle:Registration:register',
+            )));
 
-            <route id="account_register" path="/register">
-                <default key="_controller">AcmeAccountBundle:Account:register</default>
-            </route>
+            return $collection;
 
-            <route id="account_create" path="/register/create">
-                <default key="_controller">AcmeAccountBundle:Account:create</default>
-            </route>
-        </routes>
+Next, create the template:
 
-    .. code-block:: php
+.. code-block:: html+jinja
 
-        // src/Acme/AccountBundle/Resources/config/routing.php
-        use Symfony\Component\Routing\RouteCollection;
-        use Symfony\Component\Routing\Route;
+    {# app/Resources/views/registration/register.html.twig #}
+    
+    {{ form_start(form) }}
+        {{ form_row('form.username') }}
+        {{ form_row('form.email') }}
 
-        $collection = new RouteCollection();
-        $collection->add('account_register', new Route('/register', array(
-            '_controller' => 'AcmeAccountBundle:Account:register',
-        )));
-        $collection->add('account_create', new Route('/register/create', array(
-            '_controller' => 'AcmeAccountBundle:Account:create',
-        )));
+        {{ form_row('form.plainPassword.first', {
+            'label': 'Password'
+        }) }}
+        {{ form_row('form.plainPassword.second', {
+            'label': 'Repeat Password'
+        }) }}
 
-        return $collection;
+        <button type="submit">Register!</button>
+    {{ form_end(form) }}
 
 Update your Database Schema
 ---------------------------
 
-Of course, since you've added a ``User`` entity during this tutorial, make
-sure that your database schema has been updated properly:
+If you've updated the ``User`` entity during this tutorial, make sure that
+your database schema has been updated properly:
```

Comment:
Although this phrase is perfectly correct, I find it a bit confussing. Currently it can be interpreted as: *"somehow, your DB schema could have been updated, but execute this command just in case"*. But you want to say: *"if you have changed your User entity, the DB is out of sync. Execute this command to update the User table structure in the database to avoid issues"*.